.github/workflows/build-plugin-zip.yml

```
@@ -211,6 +211,28 @@ jobs:
                   name: release-notes
                   path: ./release-notes.txt
 
+    create-playground-link:
+        name: Create WordPress Playground Link
+        runs-on: ubuntu-latest
+        needs: build
+        if: |
+            always() && (
```

Minor, but I think the _unless cancelled_ trigger would be more suitable than _[always](https://docs.github.com/en/actions/learn-github-actions/expressions#always)_ here:

```suggestion
            ${{ !cancelled() }} && (
```

---

Thanks for the suggestion! There's a small chance it might run even after it is canceled if the build has finished successfully before, so that makes sense! I actually struggled to make it run without the `always` call there, it's quite unintuitive.